girder/utility/plugin_utilities.py

```diff
@@ -122,7 +122,7 @@ def loadPlugins(plugins, root, appconf, apiRoot=None, buildDag=True):
     return root, appconf, apiRoot
 
 
-def getToposortedPlugins(plugins=None, ignoreMissing=False, keys=('dependencies',)):
+def getToposortedPlugins(plugins=None, ignoreMissing=False, keys=('dependencies',), logNew=False):
```

Comment:
Agreed, I think we should not clutter the business logic API with logging options. I think the best practice is to log everything (sending it to `DEBUG` level if you're concerned about spam under the default configuration) then control the output via filters in handlers.